apps/api/src/app/testing/billing/quota-throttler.guard.e2e-ee.ts

```diff
@@ -1,8 +1,8 @@
 import { UserSession } from '@novu/testing';
 import { expect } from 'chai';
 import sinon from 'sinon';
-import { EvaluateEventResourceLimit } from '@novu/ee-billing';
 import { ApiServiceLevelEnum } from '@novu/shared';
+import { GetEventResourceLimit } from 'enterprise/packages/billing/dist';
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is this new absolute import path a mistake? We should prefer to use the NPM package name import.‏